poster/poster.tex

```diff
@@ -145,40 +145,52 @@
 
 \section*{Abstract}
 {\tt stginga}\cite{stginga} is an image visualization package to assist
-in data analysis and quality assurance of science data from the Hubble
-and James Webb space telescopes.  It is based on the
-{\tt ginga}\cite{ginga} toolkit for building scientific viewers.
-In this poster we will describe the main plugins developed for data
-analysis and quality assurance tasks with stginga.  We also discuss the
-basic outline of writing a ginga plugin, with pointers to documentation
+in data analysis and quality assurance of science data from Hubble Space
+Telescope (HST) and James Webb Space Telescope (JWST).  It is based on the
+Ginga\cite{ginga} toolkit for building scientific viewers.
+In this poster, we will describe the main plugins developed for data
+analysis and quality assurance tasks with {\tt stginga}.  We also discuss the
+basic outline of writing a Ginga plugin, with pointers to documentation
 and examples.
 
 \section*{Introduction}
-{\tt ginga}\cite{Jeschke15A} is a Python package that
+
+\begin{center}
+\includegraphics[width=8in]{ref_viewer}
+\label{fig:refviewer}
+\vspace*{0.4em}
+{\small\em Figure 1: The Ginga reference viewer.}
+\end{center}
+
+\para
+Ginga\cite{Jeschke15A} is a Python package that
 implements a toolkit for building scientific viewers.  It provides
-a \emph{reference viewer} (see Fig. \ref{fig:ref-viewer}) which features
```

Comment:
Note: This doesn't work for me, so I hardcoded the Fig num in text. Looks like it is already hardcoded in caption anyway.